aws-rds-cfn-common/src/test/java/software/amazon/rds/common/test/AbstractTestBaseTest.java

```diff
@@ -162,11 +162,10 @@ void test_handleRequest_base_NoSupplyExpect() {
 
     @Test
     public void test_randomString() {
-        final TestAbstractTestBase testBase = new TestAbstractTestBase();
         final int length = 16;
         final String alphabet = "abc";
 
-        final String randStr = testBase.randomString(length, alphabet);
+        final String randStr = AbstractTestBase.randomString(length, alphabet);
```

Comment:
I believe we should stop promoting imports from an abstract class. It sounds like we're pending a refactoring.